linux/shim/exec.go

```diff
@@ -86,7 +89,7 @@ func (e *execProcess) ExitedAt() time.Time {
 	return e.exited
 }
 
-func (e *execProcess) SetExited(status int) {
+func (e *execProcess) setExited(status int) {
```

Comment:
Do we still need the internal mutex `mu`?
Will there be race between `ExitStatus`, `ExitedAt` and `setExited`?

`status` might be fine, because it's a primitive type, not sure how golang will deal with it underlying. But `exited` is a data structure, I feel like we may get corrupted data without lock around it.

And the same question for init.go

Reply:
updated to lock around exit status and exited at